test/ext/new-element.js

```diff
@@ -0,0 +1,11 @@
+// From https://github.com/npm-dom/new-element
+
+var domify = require("domify");
```

Comment:
Use `const` instead of `var` here. Just a style thing. We should never be altering module objects.